src/ReTestItems.jl

```diff
@@ -174,7 +175,7 @@ function runtests(
     nworkers::Int=parse(Int, get(ENV, "RETESTITEMS_NWORKERS", "0")),
     nworker_threads::Union{Int,String}=get(ENV, "RETESTITEMS_NWORKER_THREADS", "2"),
     worker_init_expr::Expr=Expr(:block),
-    testitem_timeout::Real=DEFAULT_TEST_ITEM_TIMEOUT,
+    testitem_timeout::Real=parse(Float64, get(ENV, "RETESTITEMS_TIMEOUT", string(DEFAULT_TEST_ITEM_TIMEOUT))),
```

Comment:
i've always found `testitem_timeout` a verbose name, but i presumed it was because we wanted to reserve space for a possible "global" `timeout` (i.e. all tests in total must finish in X seconds, not just each individual test item)...  

if that's the case, then we should probably do the same with the ENV var, and have it be named something like `RETESTITEMS_TESTITEM_TIMEOUT`

I also think matching the keyword is good for providing a consistent interface

(and we probably want to edit the internal const to be spelled `DEFAULT_TESTITEM_TIMEOUT` for consistency)

```suggestion
    testitem_timeout::Real=parse(Float64, get(ENV, "RETESTITEMS_TESTITEM_TIMEOUT", string(DEFAULT_TESTITEM_TIMEOUT))),
```